sentry/src/main/java/io/sentry/event/interfaces/ExceptionMechanism.java

```diff
@@ -0,0 +1,71 @@
+package io.sentry.event.interfaces;
+
+import java.io.Serializable;
+
+/**
+ * Class describing an exception's mechanism.
+ */
+public final class ExceptionMechanism implements Serializable {
```

Comment:
I am afraid this class name does not directly suggest the purpose of this class.
Maybe `ExceptionSource` would be a better fit?

Reply:
The name is after Sentry's Protocol. I'm afraid I can't change that.